src/main/org/nlogo/window/SliderPainter.java

```diff
@@ -9,8 +9,6 @@
 
   abstract public java.awt.Dimension getMaximumSize();
 
-  abstract public void setToolTipText(String text);
-
```

Comment:
When this is present, the full variable name shows when you put your mouse over the handler and channel. I think it's more appropriate to show the variable name only when you actually hover the variable name, not when you change the value.

This also makes the tooltip move around every time you move your mouse between the various components(namely handler, channel and the name).